src/bin/nydusd/fs_cache.rs

```diff
@@ -538,11 +573,26 @@ impl FsCacheHandler {
                     }
                     Ok(md) => {
                         let cache_file = unsafe { File::from_raw_fd(msg.fd as RawFd) };
+                        let bootstrap_fd = f.as_raw_fd();
+                        let cachefile_fd = cache_file.as_raw_fd();
                         let object = FsCacheObject::Bootstrap(Arc::new(FsCacheBootStrap {
                             bootstrap_file: f,
                             cache_file,
                         }));
                         e.insert((object, msg.fd));
+                        let len = md.len() as usize;
+                        ASYNC_RUNTIME.spawn_blocking(move || {
+                            //add slight delay to let copen reply first
+                            thread::sleep(time::Duration::from_millis(10));
```

Comment:
Is the 10ms delay reliable? I mean if copen reaches the kernel after 10ms eclipse. What will happen. If we can bear the result, better to add a comment to tell its safety.

Reply:
if we write the cachefile anon fd before reply copen then will get eio.  In this case we will retry fill bootstrap 2s later